specs/macos/mac_mouse_acceleration.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# The name of the workflow.
+name: MacOS 13+ - Disable/Enabled mouse acceleration
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: |-
+  echo "old value:"
+  defaults read -g com.apple.mouse.scaling 
+  defaults write -g com.apple.mouse.scaling -integer {{acceleration_amount}}
+  echo "new value:"
```

Comment:
hey @mikikiv Thanks for submitting this PR!

Could you omit echoing the new value, since the developer knows the new value when they set it?